perfmetrics/scripts/custom_vm_perf_test/custom_vm_perf_script_test.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2024 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# unit tests for custom_vm_perf_script
+
+import unittest
+from custom_vm_perf_script import _parse_arguments, DEFAULT_VM_NAME, \
```

Comment:
from custom_vm_perf_script import custom_vm_perf_script

then use all the things which you want to use with `custom_vm_perf_script.`
e.g.
custom_vm_perf_script._parse_arguments
custom_vm_perf_script.DEFAULT_VM_NAME